app/selectors.js

```diff
@@ -593,9 +603,14 @@ export const transactionNormalizer = createSelector(
         const amount = credit.getAmount();
         const address = credit.getAddress();
         const creditedAccount = credit.getAccount();
-        creditedAccountName = getAccountName(creditedAccount);
+        const currentCreditedAccountName = getAccountName(creditedAccount);
+        // If we find crediton account differnt than default, then we pick
+        // it as receiver
+        if (currentCreditedAccountName !== "default" || !creditedAccountName) {
```

Comment:
I believe this shouldn't be hard coded for default. Changing the account name, from default -> "whatever" would break it, or using a different account

Reply:
updated